src/Entity/CitizenProject.php

```diff
@@ -131,7 +134,7 @@ public function __construct(
         string $name,
         string $subtitle,
         CitizenProjectCategory $category,
-        ?Committee $committee,
+        ?array $committees,
```

Comment:
Do we still expect `null` here (and not an empty array)?

Reply:
✔️